example/src/components/videoplayer/VideoPlayer.tsx

```diff
@@ -190,6 +200,43 @@ export class VideoPlayer extends PureComponent<VideoPlayerProps, VideoPlayerStat
     console.log(TAG, 'onAdEvent', type, ad);
   };
 
+  private onCastStateChangeEvent = (data: ChromecastChangeEvent | AirplayStateChangeEvent) => {
+    const stateEvent = data;
+    let message = undefined;
+    const castTarget = data.type === 'chromecaststatechange' ? 'chromecast' : 'airplay';
```

Comment:
nit-pick, can't we check if "data is ChromecastChangeEvent" ?

Reply:
Unfortunately not, both `ChromecastChangeEvent` and `AirplayStateChangeEvent` are interfaces.